packages/fxa-auth-server/test/local/oauthdb.js

```diff
@@ -1,364 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-'use strict';
-
-const { assert } = require('chai');
```

Comment:
This gets covered by `remote/oauth_tests.js` and the content-server functional tests. I didn't do an exhaustive comparison because the new `callRoute` and `backendService.sendRequest` are so similar